src/test/java/games/strategy/util/VersionTest.java

```diff
@@ -74,4 +65,12 @@ public void getExactVersion() {
     assertEquals("1.2.3.4", new Version(1, 2, 3, 4).getExactVersion());
     assertEquals("1.2.3.4.5", new Version("1.2.3.4.5").getExactVersion());
   }
+
+  @Test
+  public void testCompatible() {
+    assertTrue(new Version(1, 9).isCompatible(new Version(1, 9)));
+    assertFalse(new Version(2, 9).isCompatible(new Version(1, 9)));
+    assertFalse(new Version(1, 10).isCompatible(new Version(1, 9)));
+    assertFalse(new Version(2, 10).isCompatible(new Version(1, 9)));
```

Comment:
Would it be appropriate to add some tests that demonstrate `m_point` and `m_micro` are not considered by the algorithm?  For example (untested):

```java
assertTrue(new Version(1, 9, 0).isCompatible(new Version(1, 9, 1)));
```

Reply:
Good Point